Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
Well, the big problem is that this means that to use the latest `skalibs` the `execline` formula needs to be rebuilt every time. I think what would make more sense in this case is we remove these libraries from Homebrew and use `resource`s on the software that needs them (which seems to be just `s6` at this point, possibly ever).